docs/src/tool/index.md

```diff
@@ -292,55 +321,56 @@ development API key), and sets `REACH_RPC_TLS_REJECT_UNVERIFIED` to
 Consider this example from the @{seclink("tut-7-rpc")} tutorial:
 
 ```cmd
-$ reach rpc-run python3 -u ./index.py
+$ ./reach rpc-run python3 -u ./index.py
 ```
 
 # {#ref-usage-docker-reset} `reach docker-reset`
 
-You can easily kill and remove all Docker containers by executing
+This command removes all containers and volumes from Docker. If you have any personal containers, those containers will also be removed. Containers can then be restarted in your Docker Images list. It does not uninstall any Docker images.
 
 ```cmd
-$ reach docker-reset
+$ ./reach docker-reset
 ```
 
 This can be a useful thing to try if your Docker containers stop responding to requests or otherwise misbehave, or if you have updated your Reach images (with `reach update`) but those changes are not taking effect.
-This command is a loose approximation of "turning Docker off and on again."
 It will affect all Docker containers on your machine, not just those created by Reach.
 
 # {#ref-usage-upgrade} `reach upgrade`
 
-You can upgrade your Reach installation by executing
+This command downloads the most recent `reachsh/reach-cli` Docker image only.
 
 ```cmd
-$ reach upgrade
+$ ./reach upgrade
 ```
 
-This may change the default version used by `reach` commands.
+This might change the default version used by `reach` commands.
 
 # {#ref-usage-update} `reach update`
 
-You can update the Docker images used by your Reach installation by executing
+This command downloads and installs the most recent versions of the Reach Docker image. It does not start the Docker images once installed.
 
 ```cmd
-$ reach update
+$ ./reach update
 ```
 
-This may change the patch version used by `reach` commands.
+This might change the patch version used by `reach` commands.
 
 # {#ref-usage-version} `reach version`
 
-You can see what version of Reach you have installed by running
+Check which version of the Reach command-line tool is currently installed. This returns both the semantic version (such as `0.1.7`) and the hashes version (such as [639fa565](https://hub.docker.com/layers/reachsh/reach/639fa565/images/sha256-e72fbb183e559a6f531302843c1d4debb499c9286e0ca4839ae66023c7ba2296?context=explore)). This is helpful when making sure an upgrade to a new version is complete.
```

Comment:
Where are you seeing a hash when you run this?

Reply:
I didn't this time after you mention this, but when I ran it right before writing this, it displayed both. Not sure why...I will change it.